integration/src/main/resources/configs/resiliency/DeleteInitialPods.json

```diff
@@ -13,10 +13,10 @@
   "testScripts": [
     {
       "name": "ServiceStatus",
-      "parameters": [120],
+      "parameters": [150],
```

Comment:
Nit: why did this number increase?

Reply:
To allow more time for Kubernetes to get pods ready. A pod is ready when all application containers that support a service are in ready state. This number should be determined by app services as to what an acceptable amount of time should be for a pod to recover after some unexpected crash. I increase it arbitrarily to 150 seconds here. On average I noticed it took around 45 seconds to recover from a pod crash so 150 seconds should give enough time for 2 pods to recover from back-to-back crash.